### PR TITLE
refactor bencher to track storage changes

### PR DESCRIPTION
**Pull request by ermalkaleci**

### CODE REVIEW

**Comment by xlc** [comment]
need a merge master

File: bencher/test/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		pallet_prelude::*,
+		transactional,
+	};
+	use frame_system::pallet_prelude::*;
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {}
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {}
```
Comment (ferrell-code) [inline bencher/test/src/lib.rs:22]: No need for empty hooks and config fields... Just semantics not important at all

**Review by ferrell-code — COMMENTED** [review]
Add `bencher/test` to root Cargo.toml workspace

**Review by ferrell-code — APPROVED** [review]

**Comment by ferrell-code** [comment]
Awesome bencher update!

**Pull request merged**